src/TestFramework/Coverage/JUnit/JUnitTestCaseSorter.php

```diff
@@ -59,11 +59,16 @@ public function getUniqueSortedFileNames(array $tests): iterable
             /** @var TestLocation $testLocation */
             $testLocation = current($uniqueTestLocations);
 
+            /*
+             * TestLocation gets its file path and timings from TestFileTimeData.
+             * Path for TestFileTimeData is not optional. It is never a null.
+             * Therefore we don't need to make any type checks here.
+             */
+
+            /** @var string $filePath */
             $filePath = $testLocation->getFilePath();
 
-            if ($filePath !== null) {
-                yield $filePath;
-            }
+            yield $filePath;
```

Comment:
LGTM, can you also update the return type?  `@return iterable<string>`

Reply:
Done